rules/windows/sysmon/sysmon_in_memory_assembly_execution.yml

```diff
@@ -0,0 +1,36 @@
+title: Suspicious In-Memory Module Execution
+description: Detects the access to processes by other suspicious processes which have reflectively loaded libraries in their memory space. An example is SilentTrinity C2 behaviour. Generally speaking, when Sysmon EventID 10 cannot reference a stack call to a dll loaded from disk (the standard way), it will display "UNKNOWN" as the module name. Usually this means the stack call points to a module that was reflectively loaded in memory. Adding to this, it is not common to see such few calls in the stack (ntdll.dll --> kernelbase.dll --> unknown) which essentially means that most of the functions required by the process to execute certain routines are already present in memory, not requiring any calls to external libraries. The latter should also be considered suspicious. 
+status: experimental
+date: 27/10/2019
+author: Perez Diego (@darkquassar), oscd.community
+references:
+    - https://azure.microsoft.com/en-ca/blog/detecting-in-memory-attacks-with-sysmon-and-azure-security-center/
+tags:
+    - attack.privilege_escalation
+    - attack.T1055
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection_01: 
+        EventID: 10
+        CallTrace: 
```

Comment:
probably you are using regex here, so you need to put `CallTrace|re` modifier here

Reply:
hi mate, thanks for the observation, I'm not using RegEx here actually that vertical slash "|" is just part of the string of Sysmon Event 10 (it's how it separates between stack calls). The "|" should be considered a literal character and not a RegEx "OR" it it makes sense.